source/_integrations/roku.markdown

```diff
@@ -93,7 +93,16 @@ data:
 
 ## Media Player
 
-When the Home Assistant Roku integration is enabled and a Roku device has been configured, in the Home Assistant UI the Roku media player will show a listing of the installed channels, or apps, under “source”. Select one and it will attempt to launch the channel on your Roku device. This action can also be automated, but it requires you to acquire an extra piece of information; the ```appID``` for the channel specific to your Roku. Although this information is gathered by the Roku integration, at the moment it is not exposed to the end user. This item might be added in a future release. For now though, you can easily get the information yourself. All you need to do is a simple GET API call on the same network as your device.
+When the Home Assistant Roku integration is enabled and a Roku device has been configured, in the Home Assistant UI the Roku media player will show a listing of the installed channels, or apps, under “source”. Select one and it will attempt to launch the channel on your Roku device. This action can also be automated. Beginning with Home Assistant Core 0.110, channels can be launched by ```name``` or ```appID``` (in an automation, for example) using a configuration similar to the one below:
```

Comment:
Please remove the version number reference. Our documentation should match the latest release.
Adding in specific version exceptions tend to stick around needed long and make the documentation look outdated/unmaintained.